libs/workspace-manager/src/lib/ui/filesystem-navigator/filesystem-navigator-toolbar/filesystem-navigator-toolbar.component.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { FilesystemNavigatorToolbarComponent } from './filesystem-navigator-toolbar.component';
+
+describe('FilesystemNavigatorToolbarComponent', () => {
+  let component: FilesystemNavigatorToolbarComponent;
+  let fixture: ComponentFixture<FilesystemNavigatorToolbarComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      imports: [FilesystemNavigatorToolbarComponent],
+    }).compileComponents();
+
+    fixture = TestBed.createComponent(FilesystemNavigatorToolbarComponent);
+    component = fixture.componentInstance;
+    component.path = '/mock/path';
+    component.separator = '/';
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+
+  describe('ngOnChanges', () => {
+    it('should split the path input and save the parts', () => {
+      component.ngOnChanges();
```

Comment:
I think that if you'll use `fixture.componentRef.setInput(...)` instead of `component.` you won't need to manually trigger ngOnChanges